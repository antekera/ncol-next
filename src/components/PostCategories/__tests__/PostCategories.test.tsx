import React from 'react'

import { render, RenderResult } from '@testing-library/react'

import { PostCategories } from '..'

let documentBody: RenderResult

describe('PostCategories', () => {
  beforeEach(() => {
    documentBody = render(<PostCategories />)
  })

  test('should be defined', () => {
    expect(documentBody).toBeDefined()
  })
})
