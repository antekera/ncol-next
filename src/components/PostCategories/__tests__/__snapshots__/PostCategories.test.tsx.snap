// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostCategories should match snapshots 1`] = `
<a
  aria-label="sucesos"
  class="post-category-link"
  href="/categoria/sucesos"
>
  sucesos
</a>
`;
